packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -556,11 +548,16 @@ const onSaveInterventionPolicy = async () => {
 	}
 };
 
-const createNewInterventionPolicy = () => {
-	if (!model.value?.id) return;
-	showCreatePolicyModal.value = true;
-	newBlankInterventionPolicy.value.modelId = model.value.id;
-	showSaveModal.value = true;
+const resetInterventionPolicy = () => {
+	knobs.value.transientInterventionPolicy = {
+		modelId: '',
```

Comment:
You will need to define the model Id here, otherwise when you save the policy, it will not be associated with the model

Reply:
Thanks